### DISCUSSION TITLE
Searching by custom user properties/parameters

### DISCUSSION THREAD

**Discussion by kylerowemhs** [getsentry/support#33] [Support] [answered]
Hi!

I have a few custom user properties set as part of logging within my Sentry project. I noticed that when attempting to search/filter results by these custom attributes, they are unrecognized by Sentry.

Is there something that I'm doing wrong? Is this a planned feature at all? Or if it's ignored, what's the rationale behind that decision?

Thanks!

**Comment by markstory** [comment, marked as answer]
How are you adding the custom user properties? Are they on the user context or as tags?

**Reply by kylerowemhs** [reply]
They're currently on the user context. I'm able to use tags in custom search just fine. But the user context properties are not usable it seems

**Reply by markstory** [reply]
Yeah additional user fields are not indexed and exposed as searchable keys unfortunately. Generally using tags is better if you need to search on a property.
